lib/ChunkTemplate.js

```diff
@@ -12,6 +12,8 @@ function ChunkTemplate(outputOptions) {
 module.exports = ChunkTemplate;
 
 ChunkTemplate.prototype = Object.create(Template.prototype);
+ChunkTemplate.prototype.constructor = ChunkTemplate;
```

Comment:
why? This common style of class definition should not cause error? May be some hidden bug?
Sorry, when i patched this, i didn't test these.

I'll investigate it soon